projects/common/src/lib/controls/api-access/api-access.component.ts

```diff
@@ -68,7 +70,9 @@ export class ApiAccessComponent implements OnChanges, OnInit {
   }
 
   public ShowKey(keyData: any) {
```

Comment:
alphabetical order

Reply:
I didn't add this property, so I didn't notice that.